networklibrary/src/main/java/com/eis/smsnetwork/smsnetcommands/SMSAcceptInvite.java

```diff
@@ -0,0 +1,33 @@
+package com.eis.smsnetwork.smsnetcommands;
+
+import android.util.Log;
+
+import com.eis.communication.network.commands.CommandExecutor;
+import com.eis.smslibrary.SMSManager;
+import com.eis.smslibrary.SMSMessage;
+import com.eis.smslibrary.SMSPeer;
+import com.eis.smsnetwork.RequestType;
+import com.eis.smsnetwork.SMSJoinableNetManager;
+
+/**
+ * @author Marco Cognolato
+ * @author Giovanni Velludo
+ */
+public class SMSAcceptInvite extends com.eis.communication.network.commands.AcceptInvite<SMSPeer> {
+
+    /**
+     * Constructor for SMSAcceptInvite command, requires data to work
+     *
+     * @param inviter The SMSPeer who sent the invitation to his network
+     */
+    public SMSAcceptInvite(SMSPeer inviter) {
+        super(inviter);
+    }
+
+    protected void execute() {
+        CommandExecutor.execute(new SMSQuitNetwork(SMSJoinableNetManager.getInstance().getNetSubscriberList()));
+        SMSJoinableNetManager.getInstance().getNetSubscriberList().addSubscriber(inviter);
```

Comment:
Could have asked for a generic `JoinableNetManager` in the constructor because what if I'm not using the default SMSJoinableNetManager? I would have to re-write this command _and we don't want that_